From 75272c0762f293696605c3b5ad7833ec4954a9d5 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Wed, 20 Jun 2012 22:00:41 -0400
Subject: [PATCH] yaffs2: change end_writeback to clear_inode

commit: dbd5768f vfs: Rename end_writeback() to clear_inode()
was merged in 3.5-rc1, so yaffs2 must be updated to the new naming
scheme.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/yaffs2/yaffs_vfs.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/yaffs2/yaffs_vfs.c b/fs/yaffs2/yaffs_vfs.c
index 515bee0..a11341f 100644
--- a/fs/yaffs2/yaffs_vfs.c
+++ b/fs/yaffs2/yaffs_vfs.c
@@ -1064,7 +1064,7 @@ static void yaffs_evict_inode(struct inode *inode)
 	if (!inode->i_nlink && !is_bad_inode(inode))
 		deleteme = 1;
 	truncate_inode_pages(&inode->i_data, 0);
-	end_writeback(inode);
+	clear_inode(inode);
 
 	if (deleteme && obj) {
 		dev = obj->my_dev;
-- 
1.7.5.4

